runtime/src/main/java/com/vaadin/quarkus/QuarkusInstantiator.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.quarkus;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.spi.BeanManager;
+import javax.inject.Inject;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Stream;
+
+import io.quarkus.arc.Unremovable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.di.DefaultInstantiator;
+import com.vaadin.flow.di.Instantiator;
+import com.vaadin.flow.i18n.I18NProvider;
+import com.vaadin.flow.server.VaadinService;
+import com.vaadin.flow.server.VaadinServiceInitListener;
+import com.vaadin.quarkus.annotation.VaadinServiceEnabled;
+
+@VaadinServiceEnabled
+@Unremovable
+@ApplicationScoped
+public class QuarkusInstantiator implements Instantiator {
+
+    private static final String CANNOT_USE_CDI_BEANS_FOR_I18N = "Cannot use CDI beans for I18N, falling back to the default behavior.";
+    private static final String FALLING_BACK_TO_DEFAULT_INSTANTIATION = "Falling back to default instantiation.";
+
+    private AtomicBoolean i18NLoggingEnabled = new AtomicBoolean(true);
+    private DefaultInstantiator delegate;
+
+    @Inject
+    BeanManager beanManager;
```

Comment:
`private` ?

Reply:
It's as per Quarkus usage recommendation. Havin it as private will on runtime print:
```
[INFO] [io.quarkus.arc.processor.BeanProcessor] Found unrecommended usage of private members (use package-private instead) in application beans:
	- @Inject field com.vaadin.quarkus.QuarkusVaadinServlet#beanManager,
```

Reply:
:(